src/components/dialogs/notificationDialogUtils/NotificationIndividualMergeRequestDialog.jsx

```diff
@@ -29,9 +29,10 @@ export default function NotificationIndividualMergeDialog({
     loading: allowLoading,
   } = useAllowMerge();
 
-  const error = blockError || allowError;
   const isError = blockError || allowError;
 
+  const error = blockError ? blockError : allowError; // eslint-disable-line
```

Comment:
So get this: the formatter kept changing this to:

const error = blockError || allowError;

!!! This also explains the empty commit message I kept getting.